flamingpy/codes/graphs/stabilizer_graph.py

```diff
@@ -64,13 +68,13 @@ class StabilizerGraph(ABC):
         translating the outcomes.
     """
 
-    def __init__(self, ec, code=None):
+    def __init__(self, ec=None, code=None):
```

Comment:
Is there any reason to have `ec` and `code` optional rather than mandatory?

Reply:
You may want to build a custom stabiliser graph without providing a code. This is useful for testing amongst others.